core/src/main/java/dk/alexandra/fresco/lib/collections/sort/OddEvenMerge.java

```diff
@@ -66,7 +66,7 @@ private void merge(int first, int length, int step, ProtocolBuilderBinary builde
         }
         return null;
       });
-    } else if (length == 2) {
```

Comment:
I would say either remove it entirely if it shouldn't be there, or have it stay. No point in having old incorrect code lying around in comments.

Reply:
Agree, please dont leave code commented out. Either keep it or delete it. This is an issue a few other places